### DISCUSSION TITLE
Are there any IDE CDROM controllers that allow booting from CD?

### DISCUSSION THREAD

**Discussion by Zack13358** [86Box/86Box#3733] [Q&A] [answered]
Are there any external ISA/PCI IDE controllers that allow booting from CD if the mobo BIOS doesn't support CD boot / El Torito?

(I know there are SCSI controllers that can do this but I don't want to have to screw around with SCSI drivers)

**Comment by jriwanek** [comment, marked as answer]
That would require some sort of boot rom, I suggest instead you use a floppy with something like plop boot <https://www.plop.at/en/home.html>

**Comment by sskras** [comment]
Tried asking a similar question here, just about the physical controllers:
https://hardwarerecs.stackexchange.com/questions/18987/scsi-ide-sata-cards-for-booting-from-odd-with-widest-coverage-of-x86-mainboards

No offers so far.
